Complete Apollo 8

georgebrock opened the pull request:
This should complete Apollo 8:

* Cleaned up remaining issues with TEC
* Added _meta
* Added MEDIA
* All assets

MEDIA assets and original PDF page images aren't committed, but I do have copies locally.

https://trello.com/c/riemIKxT/11-apollo-8-21-12

jaylett commented on missions/a8/transcripts/_meta:
```diff
@@ -1,82 +1,406 @@
-    "featured": false,
-    "incomplete": true,
+    "featured": true,
+    "incomplete": false,
```
I think in future we should separate this stuff out from getting the missions done, since we can then deploy as a "hidden" mission and check there, which allows for more people to do a final pass than if they have to check out a branch and rebuild locally.

georgebrock replied:
We could revert 5bf7265e6df518952809d9b83d8387a6cefc0419, and deploy the rest of the branch?

jaylett replied:
Yes, let's!

> On 28 Jul 2015, at 18:02, George Brocklehurst <notifications@github.com> wrote:
> 
> In missions/a8/transcripts/_meta:
> 
> > @@ -1,82 +1,406 @@
> >  {
> >      "name": "a8",
> > -    "featured": false,
> > -    "incomplete": true,
> > +    "featured": true,
> > +    "incomplete": false,
> We could revert 5bf7265, and deploy the rest of the branch?
> 
> —
> Reply to this email directly or view it on GitHub.
>

georgebrock commented:
@jaylett I've got rid of the commit that marked the mission as complete and featured.

(And while I was there: rebased onto master, fixed the merge conflicts, and squashed some of the more similar commits together)

jaylett commented:
@georgebrock LGTM, you need a final squash then good to merge.
